CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to
 
 ### Added
 
+- Allow to filter enrollment resource by `has_been_created_with_order`
```

Comment:
I think "was" is good enough and simpler:
```suggestion
- Allow to filter enrollment resource by `was_created_via_order`
```
or by order?

Reply:
Let's use `was_created_by_order`